docs/deployment/information.md

```diff
@@ -5,15 +5,6 @@ Merged work is automatically deployed to all environments.
 Below details at what point in the process these auto-deployments will occur.
 
 * dev + staging + sandbox = immediately after code is merged and passes ci
-* production = once a day at 1pm CT
+* production = once a day at 2pm ET
```

Comment:
I thought it was 3pm ET. No?

Reply:
Yes thank you! This hasn't been updated in a long time.

Reply:
Not a blocking comment, but I realized you could probably remove this and just add a link to this page: https://depo-platform-documentation.scrollhelp.site/developer-docs/deployments